wiki/fr/fr-Getting-Started.md

```diff
@@ -0,0 +1,68 @@
+
+## Tout est bon ?
+
+Si vous voulez configurer Jamulus, visitez [la section matériel](Hardware-Setup).
+
+Pour de plus amples informations à propos de l'utilisation de Jamulus, regardez [le mode d'emploi](https://github.com/corrados/jamulus/blob/master/src/res/homepage/manual.md).
+
+## D'autres guides
+* [Cet excellent guide (en)](https://www.facebook.com/notes/jamulus-online-musicianssingers-jamming/idiots-guide-to-jamulus-app/510044532903831/){: target="_blank" rel="noopener noreferrer"} par [Simon Tomlinson](https://www.facebook.com/simon.james.tomlinson?eid=ARBQoY3KcZAtS3pGdLJuqvQTeRSOo4gHdQZT7nNzOt1oPMGgZ4_3GERe-rOyH5PxsSHVYYXjWwcqd71a){: target="_blank" rel="noopener noreferrer"} (_Facebook_) pourrait aussi vous aider.
+
+## Un ou des problème(s) ? Ça ne fonctionne pas longtemps ?
+
+**ÉCOUTEZ. UNIQUEMENT. LE. SIGNAL. DE. VOTRE. SERVEUR !**
+
+Ou dit autrement : n'écoutez **pas** votre signal directement.
+
+C'est évidemment compliqué pour les chanteurs et pour certains instruments, mais même si votre connexion est excellente, si les chanteurs/musiciens n'écoutent pas leur propre son provenant du serveur, votre synchronisation en pâtira. (Attention : cela n'a rien à voir avec les commandes « mutes » de Jamulus)
+
+Vous trouverez plus d'informations sur comment éviter de vous écouter (et pour vous aider à résoudre d'autres problèmes) dans la [section FAQ - Dépannage](Client-Troubleshooting) (_actuellement en construction_)
+
+## Comment Jamulus fonctionne (principe de base)
+
+**Voici comment des millions de personne utilisent Jamulus pour jouer, chanter et apprendre la musique avec d'autres personnes en temps réel à travers le monde :**
+
+![Un schéma pour expliquer le principe de fonctionnement de Jamulus](https://user-images.githubusercontent.com/44293594/99189852-b2351180-2763-11eb-96fc-660f5038a83c.png)
+_Jamulus fonctionne sur le principe serveur-clients. Le son de chacun est envoyé au serveur, qui les mixe. Ensuite, le son est envoyé à chaque client. Si un serveur est public et enregistré sur un serveur central, ses informations seront diffusés à tous les clients._
+
+## Notes
+[^1]: [Tous les détails ici](Network-Requirements){: target="_blank" rel="noopener noreferrer"}. Si vous avez une connexion avec 10Mbits descendants et 1Mbps montants, vous n'aurez sûrement aucun problème de bande-passante.
+[^2]: Les écouteurs Bluetooth basiques ont une latence trop importante, c'est la raison pour laquelle des écouteurs filaires sont recommandés. C'est aussi une des raisons pour laquelle les micros filaires sont recommandés. Particulièrement si vous chantez ou jouez d'un instrument accoustique, afin d'être sûr d'être synchronisé avec les autres. (regardez cette [notice](Getting-Started#having-trouble-cant-keep-in-time) pour plus d'informations).
```

Comment:
« Les casques ou écouteurs Bluetooth »

I had proposed that in #41, the original doc doesn't talk about wired microphones. On the other hand no latency with HF systems ;) (I'm a sound engineer, among others...)

```md
[^2]: Les casques sans fil Bluetooth ont généralement beaucoup de latence. C'est pourquoi il est préférable d'utiliser un casque ou des écouteurs filaires. En particulier, si vous chantez ou jouez d'un instrument acoustique, pour être sûr de jouer en même temps, n'utilisez qu'un casque ou des écouteurs filaires (voir [ces notes](Getting-Started#having-trouble-cant-keep-in-time) pour de plus amples informations).
```

Reply:
> ```md
> Getting-Started#having-trouble-cant-keep-in-time
> ```

we will have to change this after after the translation of the linked pages.

Reply:
> couldn't work (to be tested)
> If not wok associated title must be modified and the links too.

I don't know